d2l-audio.serge.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "d2l-audio",
+  "source_dir": "locales/json",
+  "output_file_path": "locales/json/%LANG%.json",
+  "output_lang_rewrite": [
+    "ar-sa ar",
+    "de-de de",
+    "es-mx es",
+    "fr-ca fr",
+    "ja-jp ja",
+    "ko-kr ko",
+    "nl-nl nl",
+    "pt-br pt",
+    "sv-se sv",
+    "tr-tr tr",
+    "zh-cn zh-CN",
+    "zh-tw zh-TW"
```

Here's the most up-to-date `rewrite` we should be using:
```suggestion
    "ar-sa ar",
    "cy-gb cy",
    "da-dk da",
    "de-de de",
    "es-mx es",
    "fr-ca fr",
    "ja-jp ja",
    "ko-kr ko",
    "nl-nl nl",
    "pt-br pt",
    "sv-se sv",
    "tr-tr tr"
```

---

Depending on the specifics of this repo, you may want to keep `zh-cn` and `zh-tw` the way they are. Just be sure not to have anything rewrite to a base `zh` file.